tensorflow/lite/toco/graph_transformations/tests/BUILD

```diff
@@ -41,6 +41,19 @@ tf_cc_test(
     ],
 )
 
+tf_cc_test(
+    name = "idetentify_prelu_test",
```

Comment:
"identity" -- is it a typo? (also applied to the filename)

Reply:
@miaout17 , thanks for the comments, i have updated the code as per your suggestions.

Regards
Amit